mathesar/tests/views/api/test_table_api.py

```diff
@@ -102,6 +102,127 @@ def test_table_list_filter(create_table, client):
         check_table_response(response_table, table, table_name)
 
 
+def test_table_list_filter_schema(create_table, client):
+    tables = {
+        'Schema 1': create_table('Filter Schema 1', schema='Schema 1'),
+        'Schema 2': create_table('Filter Schema 2', schema='Schema 2'),
+        'Schema 3': create_table('Filter Schema 3', schema='Schema 3')
+    }
+
+    filter_tables = ['Schema 2', 'Schema 3']
+    query_str = ','.join(filter_tables)
+
+    response = client.get(f'/api/v0/tables/?schema={query_str}')
+    response_data = response.json()
+    assert response.status_code == 200
+    assert response_data['count'] == 2
+    assert len(response_data['results']) == 2
+
+    response_tables = {Schema.objects.get(id=res['schema']).name: res
+                       for res in response_data['results']}
+    for schema_name in filter_tables:
+        assert schema_name in response_tables
+        table = tables[schema_name]
+        response_table = response_tables[schema_name]
+        check_table_response(response_table, table, table.name)
+
+
+def test_table_list_filter_created(create_table, client):
+    table_name = 'Fitler Created'
+    table = create_table(table_name)
+    query_str = '2020-01-01 8:00'
+
+    response = client.get(f'/api/v0/tables/?created_before={query_str}')
+    response_data = response.json()
+    assert response.status_code == 200
```

Comment:
Since a lot of these assertions are the same, it might be worth pulling them into a separate method so you don't repeat yourself.